tools/pika_exporter/README.md

```diff
@@ -41,136 +43,138 @@ Run prometheus:
+| 36            | rocksdb.total-sst-files-size            | Total size (in bytes) of all SST files.Note: If there are too many files, it may slow down the online query.                                                                                                          |
+| 37            | rocksdb.live-sst-files-size             | Total size in bytes of all SST files belonging to the latest LSM tree.                                                                                                                                                |
+| 47            | rocksdb.block-cache-capacity            | The capacity of the block cache.                                                                                                                                                                                      |
+| 48            | rocksdb.block-cache-usage               | Memory size occupied by entries in the block cache.                                                                                                                                                                   |
+| 49            | rocksdb.block-cache-pinned-usage        | Memory size occupied by pinned entries in the block cache.                                                                                                                                                            |
+| 51            | rocksdb.num-blob-files                  | The number of blob files in the current version.                                                                                                                                                                      |
+| 52            | rocksdb.blob-stats                      | The total and size of all blob files, and the total amount of garbage (in bytes) in blob files in the current version.                                                                                                |
+| 53            | rocksdb.total-blob-file-size            | The total size of all blob files across all versions.                                                                                                                                                                 |
+| 54            | rocksdb.live-blob-file-size             | The total size of all blob files in the current version.                                                                                                                                                              |
+| 55            | rocksdb.cfstats                         | This includes some statistical information about the column family in the database.                                                                                                                                   |
```

Comment:
The serial number of `rocksdb.cfstats` is not 55, but 5. This serial number comes from the order in the `struct Properties` of RocksDB. The RocksDB indicator that has been exposed before is not all. I marked the existing number in order, so that it can be referenced when modified.

serial number | Properties
-- | --
1 | kNumFilesAtLevelPrefix
2 | kCompressionRatioAtLevelPrefix
3 | kStats
4 | kSSTables
5 | kCFStats
6 | kCFStatsNoFileHistogram
7 | kCFFileHistogram
8 | kDBStats
... | ...